airbyte-integrations/connectors/source-netsuite/source_netsuite/streams.py

```diff
@@ -0,0 +1,209 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+
+from collections import OrderedDict
+from typing import Any, Iterable, List, Mapping, MutableMapping, Optional
+
+# Basic full refresh stream
+import requests
+import zeep
+from airbyte_cdk.models import SyncMode
+from airbyte_cdk.sources.streams.http import HttpStream
+from netsuitesdk import NetSuiteConnection
+
+
+class NetsuiteSoapStream(HttpStream):
+
+    def __init__(self, nc: NetSuiteConnection, config: dict, concurrency_limit: int = 1):
```

Comment:
This is not needed, if it's a constant of 1. Is there any reason of declaring it?